lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -458,7 +458,7 @@ Block *CircuitOp::getBodyBlock() { return &getBody().front(); }
 
 static SmallVector<PortInfo> getPortImpl(FModuleLike module) {
   SmallVector<PortInfo> results;
-  for (unsigned i = 0, e = getNumPorts(module); i < e; ++i) {
+  for (unsigned i = 0, e = module.getPortNames().size(); i < e; ++i) {
```

Comment:
(This is temporary, right? Or could `FModuleLike` have a `getNumPorts()`?)

Same Q re:whether can walk these together, but haven't chased down the underlying representations.